companion/HELP.md

```diff
@@ -13,30 +13,30 @@ Much of the general behavior is configured here. Parts of the configuration can
   This can be used to specify how the formatted output should be formatted.  
 	There are a few different possibilities how formatting can be done:
 	- None  
     Easy, there will be no formatting which gives the best performance obviously by just using the same result for formatted as raw.
 	- ECMA-376  
     When you type data in a cell of your favorite spreadsheet application, it will try to format it for you. There are two parts to it, first it will try to detect what kind of data you are entering and then it will format it to your liking. Usually you can adjust the formatting with a magical syntax and that syntax is standardised in ECMA-376.  
-		You can use it with this module as well. If you choose 'automatic' it will try to automatically detect the type of the entered data as text, a number, a date or a time. If you don't want to have this guesswork, you can also deliberately choose one of the data types. Second you can set up how the data should be presented, exactly the way like you are used from spreadsheet apps. E.g. 0000.00 will always show a number with at least four digits and rounded to two decimals. [Here](https://support.microsoft.com/en-us/office/review-guidelines-for-customizing-a-number-format-c0a1d1fa-d3f4-4018-96b7-9c9354dd99f5) is the full documentation.  
-		This module also allows to mess with locales. If you add a locale string in curly brackets like 0.00{de} the output will be 0,00 instead of 0.00 as the decimal separator in Germany is the comma. Locales so far only are working for the formatting, not for the interpreting. That means if you want to enter a date you have to do it in US notation like mm/dd/yy but then you can format it in your locale. Supported locales are: cs, da, de, el, en, es, fi, fr, hu, is, id, it, ja, ko, nb, nl, pl, pt, ru, sk, sv, th, tr, zh.
+		You can use it with this module as well. If you choose 'automatic' it will try to automatically detect the type of the entered data as text, a number, a date or a time. If you don't want to have this guesswork, you can also deliberately choose one of the data types. Second you can set up how the data should be presented, exactly the way you are used to from spreadsheet apps. E.g. 0000.00 will always show a number with at least four digits and rounded to two decimals. [Here](https://support.microsoft.com/en-us/office/review-guidelines-for-customizing-a-number-format-c0a1d1fa-d3f4-4018-96b7-9c9354dd99f5) is the full documentation.  
+		This module also allows you to mess with locales. If you add a locale string in curly brackets like 0.00{de} the output will be 0,00 instead of 0.00 as the decimal separator in Germany is the comma. Locales so far only are working for the formatting, not for the interpreting. That means if you want to enter a date you have to do it in US notation like mm/dd/yy but then you can format it in your locale. Supported locales are: cs, da, de, el, en, es, fi, fr, hu, is, id, it, ja, ko, nb, nl, pl, pt, ru, sk, sv, th, tr, zh.
```

Comment:
From the code it looks like en-US and en-GB are probably supported, but aren't listed.

Reply:
Any opinion on the locales @dnmeid to improve the docs further?

Reply:
There is no support for en-US and en-GB, the given locales are not supporting variants to keep the footprint small. The en locale is giving en-US month names. So far nobody has reached out to me with a request for finer grained locales, I guess it's fine for a gentleman to read October instead of Octobre.

Reply:
Ah, I was thrown because the regex matched them, and the comment in the help file "That means if you want to enter a date you have to do it in US notation like mm/dd/yy but then you can format it in your locale." but it doesn't look like the locales drive any date based formatting in that sense, so I guess it doesn't really matter.

They can always use "fr" if they want Octobre; I'm not aware of any language barrier between British and American, aside from whether it's day/month or month/day.